backend/codesirius/problems/serializers/execution_constraint_v2.py

```
@@ -0,0 +1,203 @@
+            logger.info("Bulk upserting execution constraints")
+            constraints = [
+                ExecutionConstraint(**data, problem=self.context["problem"])
+                for data in validated_data
+            ]
+            return self.child.Meta.model.objects.bulk_create(
+                constraints,
+                update_conflicts=True,
+                unique_fields=["problem", "language"],
+                update_fields=["time_limit", "memory_limit"],
+            )
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)